docs/nddata/nddata.rst

```diff
@@ -171,3 +204,8 @@ array like this::
     >>> input_array = np.ma.array([1, 2, 3, 4], mask=[1, 0, 0, 1])
     >>> ndd_masked = NDData(input_array)
     >>> masked_array = np.ma.array(ndd_masked.data, mask=ndd_masked.mask)
+
+The same applies to extracting the data as Quantity::
+
+    >>> ndd_quantity = NDData([1, 2, 3, 4], unit="meter")
+    >>> quantity = u.Quantity(ndd_quantity, unit=ndd_quantity.unit)
```

Comment:
this throws a TypeError for me. Also I wonder why one should add the nddata's unit as unit separately, shouldn't it be recognized by default?

```
UNEXPECTED EXCEPTION: TypeError(u'The value must be a valid Python or Numpy numeric type.',)
Traceback (most recent call last):

  File "/sw/lib/python2.7/doctest.py", line 1315, in __run
    compileflags, 1) in test.globs

  File "<doctest nddata.rst[34]>", line 1, in <module>

  File "astropy/units/quantity.py", line 264, in __new__
    raise TypeError("The value must be a valid Python or "

TypeError: The value must be a valid Python or Numpy numeric type.
```